src/couch_replicator/src/couch_replicator.erl

```diff
@@ -182,6 +115,8 @@ rep_result_listener(RepId) ->
         end).
 
 
+-spec wait_for_result(rep_id()) ->
+    {ok, any()} | {error, any()}.
```

Comment:
It doesn't seem to complain about that argument. But the type correct, it could be `{ok, {[_]}}` so I'll change it